From de47f6323d8fb20feefee21d0195cf0529151e04 Mon Sep 17 00:00:00 2001
From: Steve Dickson <steved@redhat.com>
Date: Thu, 17 Sep 2015 15:57:35 -0400
Subject: [PATCH 1/4] security.c: removed warning

src/security.c:100:8: warning: implicit declaration of function 'xlog'
[-Wimplicit-function-declaration]

Signed-off-by: Steve Dickson <steved@redhat.com>
---
 src/security.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/security.c b/src/security.c
index 0c9453f..c54ce26 100644
--- a/src/security.c
+++ b/src/security.c
@@ -17,6 +17,8 @@
 #include <syslog.h>
 #include <netdb.h>
 
+#include "xlog.h"
+
 /*
  * XXX for special case checks in check_callit.
  */
-- 
2.6.4

